### PR TITLE
tests: long function with type declaration

### PR DESCRIPTION
**Pull request by evilebottnawi**
From PSR-12: https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md#45-method-and-function-arguments

### CODE REVIEW

File: tests/class/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -341,6 +349,15 @@ abstract class ReallyReallyReallyLongClassName
     {
         return new SplQueue();
     }
+
+    public function longLongAnotherFunction(
+        string $foo,
+        string $bar,
+        int $baz
+    ): string
+    {
```
Comment (czosel) [inline tests/class/__snapshots__/jsfmt.spec.js.snap]: This is acutally a bug, isn't it?

Comment (evilebottnawi) [reply tests/class/__snapshots__/jsfmt.spec.js.snap]: @czosel yep, we have todo around this, let's do this in other PR?

Comment (czosel) [reply tests/class/__snapshots__/jsfmt.spec.js.snap]: Alright! Can you open an issue for this? (Or does one exist already?)

Comment (evilebottnawi) [reply tests/class/__snapshots__/jsfmt.spec.js.snap]: @czosel Done https://github.com/prettier/plugin-php/issues/171 :+1:

Comment (czosel) [reply tests/class/__snapshots__/jsfmt.spec.js.snap]: Thanks :)

**Pull request merged**